docker/kubernetes/templates/mysql-pvc.yaml

```
@@ -15,22 +15,21 @@
 # limitations under the License.
 #
 
-{{- if and .Values.mysql.enabled .Values.mysql.persistentVolumeClaim.enabled }}
+{{- if not .Values.external.mysql.enabled }}
+{{- if .Values.volumes.persistence }}
```

Is this style better?

```suggestion
{{- if and (not .Values.external.mysql.enabled) .Values.volumes.persistence }}
```

---

the end `{{- end }}` should be removed if using this suggestion. separate condition is clear for reading.